cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -305,47 +392,40 @@ def prompt(
                 ),
             )
 
-        return cast(
-            List[TLMResponse],
-            self._event_loop.run_until_complete(
-                self._batch_prompt(prompt, capture_exceptions=False),
-            ),
+        return self._event_loop.run_until_complete(
+            self._batch_prompt(prompt, capture_exceptions=False),
         )
 
     def try_prompt(
         self,
         prompt: Sequence[str],
         /,
-    ) -> List[Optional[TLMResponse]]:
+    ) -> List[TLMResponse]:
         """
         Gets response and trustworthiness score for any batch of prompts,
-        handling any failures (errors of timeouts) by returning None in place of the failures.
+        handling any failures (errors or timeouts) by returning error messages and retryability information in place of the failures.
 
-        The list returned will have the same length as the input list, if there are any
-        failures (errors or timeout) processing some inputs, the list will contain None in place of the response.
+        The list returned will have the same length as the input list. If there are any
```

Comment:
Is this not the same sentence as what is written above?

> handling any failures (errors or timeouts) by returning error messages and retryability information in place of the failures.

I'd shorten the original description as
"Gets response and trustworthiness score for any batch of prompts handling any failures (errors or timeouts)" and then expand below how this is handled